src/main/java/com/jcabi/github/mock/MkEvent.java

```diff
@@ -112,11 +114,65 @@ public int compareTo(
         throw new UnsupportedOperationException("#compareTo()");
     }
 
+    /**
+     * Describes the event in a JSON object.
+     * @return JSON object
+     * @throws IOException If there is any I/O problem
+     * @todo #1063 When the event has a label, retrieve and include the
```

Comment:
@cvrebert please add a time estimate. e.g., for 30 minutes, do `@todo #1063:30min When the event...`